src/civis/tests/mocks.py

```diff
@@ -30,7 +31,9 @@ def create_client_mock(cache=None):
     real_client = _real_client(cache)
 
     # Prevent the client from trying to talk to the real API when autospeccing
-    with mock.patch("requests.Session", mock.MagicMock):
+    with mock.patch("requests.Session", mock.MagicMock), warnings.catch_warnings():
+        # Ignore deprecation warning from `client.default_credential`.
```

Comment:
Does this mean that deprecation warnings will be ignored if a test tries to use the deprecated `default_credential` method? Or were deprecation warnings showing up every time the mock client was instantiated?